tests/helper/helper_interactive.go

```diff
@@ -37,6 +37,9 @@ func RunInteractive(command []string, test func(*expect.Console, *bytes.Buffer))
 	cmd.Stdin = c.Tty()
 	cmd.Stdout = c.Tty()
 	cmd.Stderr = c.Tty()
+	if env != nil {
+		cmd.Env = env
```

Comment:
Are you sure the Env is empty? I would use `cmd.Env = append(cmd.Env, env)` instead

Reply:
Yes, I checked, and `Env` is empty by default.

Actually, per [the doc](https://pkg.go.dev/os/exec#Cmd):

> If Env is nil, the new process uses the current process's environment.

I assumed that, if a non-`nil` `env` is passed, it means that callers may want to have full control over the sub-process environment, and not inherit the current process one.
Unless we want to also allow to include the current process environment; in this case, we would need to append the slice returned by `os.Environ()`, I guess..
What do you think?

Reply:
Yes, you are tight, I remember I used os.Environ() here (https://github.com/redhat-developer/odo/blob/main/pkg/devfile/image/docker_compatible.go#L36) for this purpose. 
If tests are passing without getting the original environment, let's keep it as is (`cmd.Env = env`).
Thanks